### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by deepstqte**
@cldougl Could you please review, and if I forgot to include anything that we talked about in our f2f chat the other day, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,28 @@
-Conda Buildpack
-===============
+## Conda Buildpack
 
-This is a [Heroku Buildpack](https://devcenter.heroku.com/articles/buildpacks) for [Conda](http://conda.pydata.org/), the Python distribution for scientific computing by Continuum Analytics.
+This is a Buildpack for [Conda](http://conda.pydata.org/), the Python distribution for scientific computing by Continuum Analytics.
 
------------------------
+This buildpack enables the installation of binary packages through the open source [conda](http://conda.pydata.org/) application.  Conda is recognized as being core to Continuum's Anaconda Scientific Python distro but it's also at the heart of the lighter weight [Miniconda](http://conda.pydata.org/miniconda.html) distro which we use here to install _only_ the binary packages we need for our apps deployed on Heroku.
```
Comment (cldougl) [inline README.md]: should we say something like heroku and heroku-like environments basically we could make this a little more broad to include dds & other deployment options

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
-To control what binary packages are installed by conda, supply a `conda-requirements.txt` file (which can be created by capturing the output of `conda list -e` for your working conda environment). Like when using the standard buildpack for python from Heroku, you can also still supply a `requirements.txt` file for [pip](https://github.com/pypa/pip) to process.  In this way, you can install binary packages via conda for everything you can and still use pip for anything you can't.
+### Usage and files needed
+
+#### conda requirements
+
+To control what binary packages are installed by conda, supply a `conda-requirements.txt` file (which can be created by capturing the output of `conda list -e` for your working conda environment).
+
+#### pip requirements
+
+Like when using the standard buildpack for python from Heroku, you can also still supply a `requirements.txt` file for [pip](https://github.com/pypa/pip) to process.  In this way, you can install binary packages via conda for everything you can and still use pip for anything you can't.
```
Comment (cldougl) [inline README.md]: For the following sentence: 
>In this way, you can install binary packages via conda for everything you can and still use pip for anything you can't. 

I would suggest making that a little more clear: 
This allows you to install binary packages via conda when possible and still use pip for packages that are not available via conda.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,14 @@ This buildpack enables the installation of binary packages through the open sour
 
 ### Usage and files needed
 
+#### conda runtime
+
+A `conda-runtime.txt` file is needed, it should contain the version of `conda` to install, in the form of `Miniconda<2-or-3>-<conda-version>`, e.g. `Miniconda3-4.5.12`
+
+The supported runtimes are the ones in [this list](https://repo.continuum.io/miniconda/) and that are equal to or more recent than `3.18.3`
```
Comment (cldougl) [inline README.md]: I would probably write 
> that are equal to or more recent than `3.18.3`

just as:  `>= 3.18.3`

File: README.md [hunk 4]
```diff
@@ -22,3 +22,7 @@ To control what binary packages are installed by conda, supply a `conda-requirem
 
 Like when using the standard buildpack for python from Heroku, you can also still supply a `requirements.txt` file for [pip](https://github.com/pypa/pip) to process. This way, you can install binary packages via conda for everything you can and still use pip for anything you can't.
 
+#### .condarc configuration file
+
+A [`.condarc` configuration file](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html) can be added to the root of the app, this file will be detected and used to build and run the conda app. This can be useful for a number of things as described in the official conda documentation linked to earlier in this paragraph, among which [using a proxy](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#configure-conda-for-use-behind-a-proxy-server-proxy-servers) for conda installs and [using conda channels other than the default one](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#channel-locations-channels).
```
Comment (cldougl) [inline README.md]: among which include

**Review by cldougl — APPROVED** [review]
just a few wording suggestions but overall this looks good and informative! 💃 (after considering suggestions)

File: README.md (hunk 1 above)
Comment (deepstqte) [reply README.md]: Definitely, I overlooked this, will change this.

File: README.md (hunk 4 above)
Comment (deepstqte) [reply README.md]: I replaced "among which" with "including".

File: README.md (hunk 2 above)
Comment (deepstqte) [reply README.md]: Yes. I like this better. Thank you.

**Pull request merged**